labrad/node/server_config.py

```diff
@@ -0,0 +1,114 @@
+import io
+import os
+import sys
+
+from configparser import ConfigParser
+
+from labrad.util import findEnvironmentVars
+
+
+class ServerConfig(object):
```

Comment:
This could be a named tuple with `is_local` and `verson_tuple` as @properties.  That would eliminate the boilerplate init method.  Ideally python would have a "named not-tuple" that didn't implement getitem...